test/iron-overlay-behavior.html

```diff
@@ -155,6 +155,21 @@
           assert.equal(getComputedStyle(overlay).display, 'none', 'overlay starts hidden');
         });
 
+        test('_renderOpened called only after is attached', function(done) {
+          var overlay = document.createElement('test-overlay');
+          // overlay is ready at this point, but not yet attached.
+          var spy = sinon.spy(overlay, '_renderOpened');
+          // this triggers _renderOpened
+          overlay.opened = true;
+          // Even if not attached yet, overlay should be the current overlay!
+          assert.equal(overlay, overlay._manager.currentOverlay(), 'currentOverlay ok');
+          // wait for requestAnimationFrame to kick.
+          overlay.async(function() {
```

Comment:
Hmm, this is a weird test. You're basically saying "this won't be called in 100ms", but what if it's called in 101ms? :(